src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -1415,87 +1568,124 @@ impl A320HydraulicBrakeComputerUnit {
-                    self.right_brake_yellow_output = Ratio::new::<ratio>(1.);
+                    self.alternate_brake_outputs.set_max_demands();
 
                     // Special case: parking brake on but yellow can't provide enough brakes: green are allowed to brake for emergency
                     if alternate_circuit.left_brake_pressure().get::<psi>()
                         < Self::MIN_PRESSURE_PARK_BRAKE_EMERGENCY
                         || alternate_circuit.right_brake_pressure().get::<psi>()
                             < Self::MIN_PRESSURE_PARK_BRAKE_EMERGENCY
                     {
-                        self.left_brake_green_output = self.left_brake_pilot_input;
-                        self.right_brake_green_output = self.right_brake_pilot_input;
+                        self.norm_brake_outputs.set_brake_demands(
+                            self.left_brake_pilot_input,
+                            self.right_brake_pilot_input,
+                        );
                     }
                 }
             }
         }
+    }
 
-        // Limiting final values
-        self.left_brake_yellow_output = self
-            .left_brake_yellow_output
-            .min(Ratio::new::<ratio>(1.))
-            .max(Ratio::new::<ratio>(0.));
-        self.right_brake_yellow_output = self
-            .right_brake_yellow_output
-            .min(Ratio::new::<ratio>(1.))
-            .max(Ratio::new::<ratio>(0.));
-        self.left_brake_green_output = self
-            .left_brake_green_output
-            .min(Ratio::new::<ratio>(1.))
-            .max(Ratio::new::<ratio>(0.));
-        self.right_brake_green_output = self
-            .right_brake_green_output
-            .min(Ratio::new::<ratio>(1.))
-            .max(Ratio::new::<ratio>(0.));
+    fn update_steering_demands(
+        &mut self,
+        lgciu1: &impl LgciuInterface,
+        engine1: &impl Engine,
+        engine2: &impl Engine,
+    ) {
+        let steer_angle_from_autopilot = Angle::new::<degree>(
+            self.autopilot_nosewheel_demand.get::<ratio>() * Self::AUTOPILOT_STEERING_INPUT_GAIN,
+        );
+
+        let steer_angle_from_pedals = if self.tiller_pedal_disconnect {
+            Angle::new::<degree>(0.)
+        } else {
+            self.pedal_input_map
+                .angle_demand_from_input_demand(self.rudder_pedal_position)
+        };
+
+        let final_steer_rudder_plus_autopilot = self.pedal_steering_limiter.angle_from_speed(
+            self.ground_speed(),
+            (steer_angle_from_pedals + steer_angle_from_autopilot)
+                .min(Angle::new::<degree>(
+                    Self::MAX_RUDDER_INPUT_INCLUDING_AUTOPILOT_DEGREE,
+                ))
+                .max(Angle::new::<degree>(
+                    -Self::MAX_RUDDER_INPUT_INCLUDING_AUTOPILOT_DEGREE,
+                )),
+        );
+
+        let steer_angle_from_tiller = self.tiller_steering_limiter.angle_from_speed(
+            self.ground_speed(),
```

Comment:
There's a bit of a shortcut here indeed, but I think this would be based on BSCU wheel speed sensor information, not on ADIRS. So that might be handled in a separate task when going into such details (autobrake speed would use same inputs I think).

Reply:
Is this something you want to note down somewhere around these lines in the code? Or is there another way you want to track this, e.g. by adding an issue to the project?

Reply:
That's worth a todo :)  There will be someday a global work to do on all those inputs to replace any of those with correct plane inputs. Maybe once FWS and arinc comunication gets in good shape!